ssms/basic_simulators/theta_processor.py

```diff
@@ -242,6 +242,11 @@ def process_theta(
             theta["t"] = np.expand_dims(theta["t"], axis=1)
             theta["a"] = np.expand_dims(theta["a"], axis=1)
 
+        if model == "poisson_race":
```

Comment:
thanks for adding in both places. eventually the modular stuff will take over but need to move a bit slowly to not break things.